### ISSUE TITLE
Séparation des mots par genre et pluriel dans Core/Strings/Words/Nouns/

### ISSUE THREAD

**Issue by b606** [Ludeon/RimWorld-fr#277]
Ce sont les mots utilisés pour créer les récits (Tales: noms et textes d'oeuvre d'arts, Rulepack: noms et textes divers des interaction entre personnages, dialogues, combat, Quête, etc)

Les mots en anglais sont dans un seul fichier, mais nous, on a besoin de les séparer par genre et pluriel pour construire des phrases correctes et se débarraser des un(e), le/la, son/sa etc sur ces mots.

La convention des noms de fichier, si `X.txt` est le nom de base :
      `X.txt `et `X_Plural.txt` : contiennent les mots masculins
      `X_Feminine.txt` et `X_FemPlural.txt` : contiennent les mots féminins

S'il n'y a pas de `X_Plural.txt`, `X_Feminine.txt` et `X_FemPlural.txt` on laisse pour l'instant tel quel. On traduit simplement le contenu (par exemple, `Core/Strings/Words/Nouns/Quests.txt`)

Par défaut, j'ai rempli vite fait avec ce qu'on avait déjà avant + quelques suppressions venant des mises à jour. Il nous faut :
- supprimer de chaque fichier le mot du mauvais genre
- mettre à jour les pluriels
- et corriger s'il y a une erreur

**Détail technique**: les fichiers qui sont là devraient déjà être actifs dans le jeu. Pour les autres prévus (AnimalBadass, les adjectifs, les Concept*.txt) je rajouterai au fur et à mesure car à chaque fois, il faut modifier les fichiers .xml où ces mots apparaissent. Et pour l'instant, il y a les QuestScript dans Royalty à terminer.

**Comment by kiliak** [comment]
c'est fait, il faudrait peut être repasser dessus pour rajouter du contenu.

sinon pour le fichier : PersonalCharacteristics.txt et ses variantes féminin/pluriel, je n'ai pas vraiment compri quoi y mettre. Le fichier d’origine semblait être un gros four tout...sans doute a reprendre quand on aura défini quoi y mettre.

**Comment by b606** [comment]
Effectivement, **PersonalCharacterisc** sont des listes de mots/sujets pour "complimenter" ou "insulter" un personnage. Du coup, c'est un peu différent, Il suffit que çà apparaisse une fois : on n'a pas besoin d'avoir toutes les versions du mot mais juste les séparer pour pouvoir regler le grammaire.

Ex : on complimente X pour ses `goûts` (Plur), ou son `goût de l'aventure` (M) pas son `goût` (tout court).
J'ai reglé les rulePack pour çà (normalement)
